template/{{app_name}}/package.json

```diff
@@ -65,7 +65,7 @@
     "postcss-loader": "^8.0.0",
     "postcss-preset-env": "^10.0.0",
     "prettier": "^3.4.2",
-    "sass": "^1.59.3",
+    "sass": "^1.77.0",
```

Comment:
How come the title says downgrade but the prior version is 1.59.3 😅 ?

Reply:
> How come the title says downgrade but the prior version is 1.59.3 😅 ?

npm is weird 😸 ? the version in the package lock file was `sass@1.83.0`. I ran `npm i sass@1.77.0` to "downgrade." 

It might be best to pin to that version, ie `"sass": "1.77.0"` since the warning start with `1.78.0`, but not sure what the best practice is there TBH